src/main/java/net/dv8tion/jda/api/entities/Member.java

```diff
@@ -675,7 +675,7 @@ default AuditableRestAction<Void> timeoutFor(@Nonnull Duration duration)
      */
     @Nonnull
     @CheckReturnValue
-    default AuditableRestAction<Void> timeoutUntil(@Nullable TemporalAccessor temporal)
+    default AuditableRestAction<Void> timeoutUntil(@Nonnull TemporalAccessor temporal)
```

Comment:
`null` can be passed here (it will remove the timeout). if you want to change the ABI then it should be probably changed in the documentation too

Reply:
It can't be null since the method throws (and the one in Guild is nonnull)